decidim-proposals/spec/forms/decidim/proposals/admin/import_participatory_text_form_spec.rb

```diff
@@ -17,12 +17,13 @@ module Admin
           }
         end
         let(:description) {}
-        let(:document_file) { Decidim::Dev.asset("participatory_text.md") }
+        let(:document) { Decidim::Dev.test_file("participatory_text.md", "text/markdown") }
```

Comment:
@isaacmg410 haves you tried uploading a markdown file from the UI? I think this is a hack that does not reflect what really happens. When uploading a markdown text file from UI I get an empty string mime type to the type validation in the form fails. I'm not sure if this is something that has changed in Decidim or it was already like this before.

Reply:
@isaacmg410 as we've talked offline, we're having different document type detection even in same Ubuntu Linux machines. We will have to accept `text/plain` and `text/markdown` mime types for Markdown documents.